RecoTracker/MkFitCore/src/Pool.h

```diff
@@ -1,5 +1,6 @@
 #ifndef RecoTracker_MkFitCore_src_Pool_h
 #define RecoTracker_MkFitCore_src_Pool_h
+#include <mm_malloc.h>
```

Comment:
is this for `_mm_malloc`?
Can it be replaced by `aligned_alloc`?

Reply:
probably yes ... there are also some other mm_mallocs around. I was wondering how come the include wasn't needed in cmssw build.